src/clj/matcher_combinators/midje.clj

```diff
@@ -86,3 +134,22 @@
          ::result/value  mismatch-val
          ::result/weight 1}))))
 
+(def match-equals
+  "match but using strict `equals` matching behavior for maps, even nested ones."
+  (match-with {clojure.lang.IPersistentMap matchers/equals}))
+
+(def match-roughly
+  "match where all numbers match if they are within the delta of their expected value"
+  (fn [delta matcher]
+    (let [func (fn [expected] (core/->PredMatcher (fn [actual]
+                                                    (core/roughly? expected actual delta))))]
+      (match-with
+       {java.lang.Integer    func
+        java.lang.Short      func
+        java.lang.Long       func
+        java.lang.Float      func
+        java.lang.Double     func
+        java.math.BigDecimal func
+        java.math.BigInteger func
+        clojure.lang.BigInt  func}
```

Comment:
eventually I'd like to try to group similar classes so that we can do things like `{num func}` instead of the huge map :point_up: